curriculum/challenges/english/01-responsive-web-design/basic-css/use-attribute-selectors-to-style-elements.english.md

```diff
@@ -8,7 +8,7 @@ forumTopicId: 301092
 
 ## Description
 <section id='description'>
-You have been given <code>id</code> or <code>class</code> attributes to elements that you wish to specifically style. These are known as ID and class selectors. There are other CSS Selectors you can use to select custom groups of elements to style.
+You have been giving <code>id</code> or <code>class</code> attributes to elements that you wish to specifically style. These are known as ID and class selectors. There are other CSS Selectors you can use to select custom groups of elements to style.
```

Comment:
@mikemorr If we are going to change it, let's use the word `adding`.  What do you think?
```suggestion
You have been adding <code>id</code> or <code>class</code> attributes to elements that you wish to specifically style. These are known as ID and class selectors. There are other CSS Selectors you can use to select custom groups of elements to style.
```

Reply:
@RandellDawson Either one seems good to me. :)

Reply:
You can click the Commit suggestion button to accept my suggestion.  It will commit that change.

Reply:
@RandellDawson I'm having trouble finding that button, sorry.

Reply:
The one circled in red.

![image](https://user-images.githubusercontent.com/5313213/69510933-833ee380-0ef3-11ea-8816-fc113943d4dd.png)

Reply:
I don't see the button there on my end. Is it because we are commenting on freeCodeCamp's copy of the repo, rather than mine? Screenshot attached.
![Screenshot from 2019-11-24 23-14-08](https://user-images.githubusercontent.com/2184268/69512006-5baa4400-0f10-11ea-87c9-4e686c3994bc.png)